ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml

```diff
@@ -5,6 +5,9 @@ mas_appws_spec:
     jdbc: "{{ mas_appws_bindings_jdbc | default( 'system' , true) }}"
     watsonstudio: system
   settings:
+    mat:
+      datamartid: "{{ cpd_wos_datamart_id }}"
```

Comment:
What if cpd_wos_datamart_id == "", from the doc update this is something related to OpenScale, but that's an optional dependency I thought, so shouldn't be only be conditionally adding this if OpenScale is used?

Also, in a fully auotmated install, how does one obtain this ID?  Does it come from OpenScale, if so we need to be able to pass this value into the role via the install pipeline when/if OpenScale is selected for install.

Reply:
Is my previous comment answer this first part of the question?

I can see you did something similar for project_id here: https://github.com/ibm-mas/ansible-devops/blob/master/ibm/mas_devops/roles/suite_app_config/tasks/determine-watson-studio-id.yml

I will work on logic for CPD_WOS_DATAMART_ID - do you want me to follow your approach?

Related to automation install. I have found that we can use: 
- Creating a service instance for Watson OpenScale programmatically
- create datamart ids using Watson OpenScale Python SDK’s